From f7f743b90aae9ece8d697a885eb24a28b3c2d488 Mon Sep 17 00:00:00 2001
From: Suresh Gupta <suresh.gupta@xilinx.com>
Date: Fri, 12 Oct 2018 15:37:06 +0530
Subject: [PATCH 0472/1852] v4l: xilinx-multi-scaler: Change maximum supported
 height and width

commit ffcdce218b3f228a06cf36645240c0914855faa8 from
https://github.com/Xilinx/linux-xlnx.git

Multi-Scaler IP support width and height up to 8192 and 4320 respectively.

Signed-off-by: Suresh Gupta <suresh.gupta@xilinx.com>
Reviewed-by: Saurabh Sengar <saurabh.singh@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-multi-scaler.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-multi-scaler.c b/drivers/media/platform/xilinx/xilinx-multi-scaler.c
index 7654f400b9d4..19614b3fc4b3 100644
--- a/drivers/media/platform/xilinx/xilinx-multi-scaler.c
+++ b/drivers/media/platform/xilinx/xilinx-multi-scaler.c
@@ -86,8 +86,8 @@
 #define XM2MSC_MIN_CHAN		1
 #define XM2MSC_MAX_CHAN		8
 
-#define XM2MSC_MAX_WIDTH	(3840)
-#define XM2MSC_MAX_HEIGHT	(2160)
+#define XM2MSC_MAX_WIDTH	(8192)
+#define XM2MSC_MAX_HEIGHT	(4320)
 #define XM2MSC_MIN_WIDTH	(64)
 #define XM2MSC_MIN_HEIGHT	(64)
 #define XM2MSC_STEP_PRECISION	(65536)
-- 
2.31.1

